lib/beaker-pe/install/pe_utils.rb

```diff
@@ -903,7 +903,7 @@ def do_higgs_install host, opts
 
           #wait for output to host['higgs_file']
           #we're all done when we find this line in the PE installation log
-          if version_is_less(options[:pe_ver] || host['pe_ver'], '2016.3')
+          if version_is_less(opts[:pe_ver] || host['pe_ver'], '2016.3')
```

Comment:
@phongdly when you put this code in in https://github.com/puppetlabs/beaker-pe/pull/34, it looks like the spec tests weren't updated.

Should it be that we use the local options passed, or do we need the global options hash and the spec tests should be updated?